docs/organizations/managing-repositories.md

```diff
@@ -69,6 +69,23 @@ To follow or unfollow repositories on Codacy:
 
     Conversely, you automatically **stop following** a repository as soon as you try accessing any page from that repository but you don't have permissions to see that repository anymore.
 
+## Repository Custom Properties Filter {: id="custom-properties-filter"}
+
+Codacy allows you to filter repositories based on **Repository Custom Properties** from GitHub. This feature helps you to categorize and filter repositories more effectively within the Codacy platform.
+
+![Repositories list filter](images/organization-manage-repos-custom-properties.png)
+
+To enable this feature, users must go through an **initial sync** between GitHub and Codacy, which ensures that the repository custom properties from GitHub are reflected in Codacy.
+
+![Repositories list sync](images/organization-manage-repos-custom-properties-sync.png)
+
+!!! note
+    For changes to **Repository Custom Properties** in GitHub to be **automatically** reflected in Codacy, users need to [accept the new permission request made by the Codacy GitHub app](https://docs.github.com/en/apps/using-github-apps/approving-updated-permissions-for-a-github-app). Accepting this permission enables automatic synchronization of any future changes to the custom properties in GitHub.
+
+    If the permission is **not accepted**, users will still be able to use Repository Custom Properties as filters in Codacy, but will need to manually trigger a sync. This can be done using the **manual sync** button available in the filter dropdown, which allows users to synchronize changes from GitHub, though the process may take longer.
+
+Once the sync is completed, you can use these custom properties as a filter to easily locate and organize repositories within Codacy.
```

Comment:
Just a proposition: 
IDK if we usually do this but we could add link how to add custom property in GH 
https://docs.github.com/en/organizations/managing-organization-settings/managing-custom-properties-for-repositories-in-your-organization#adding-custom-properties

Reply:
Makes a lot of sense to me, will update with suggesting.